axelor-sale/src/main/java/com/axelor/apps/sale/db/repo/ConfiguratorCreatorSaleRepository.java

```diff
@@ -33,6 +33,8 @@
 
 public class ConfiguratorCreatorSaleRepository extends ConfiguratorCreatorRepository {
 
+  protected static final String AXELOR_TMP_FIX_CONSTRAINT = "$AXELORTMP";
```

Comment:
The constant is duplicated in 3 classes, this approach doesn't improve a lot the maintainability. I recommend making the constant public in one of the class so that other classes can use it. As a consequence, they won't need to declare the constant anymore.

Reply:
Indeed, fixed it. Thank you for the feedback

Reply:
That's better, thank you.

However, I don't think the repo is the best location where to declare the constant because it's not related to data access. But I will let @ale-axelor give his opinion.